pkgs/tools/security/iaito/default.nix

```diff
@@ -9,60 +9,77 @@
 , qttools
 , radare2
 , wrapQtAppsHook
-, nix-update-script
 }:
 
-# TODO MacOS support.
-# TODO Build and install translations.
-
 stdenv.mkDerivation rec {
   pname = "iaito";
-  version = "5.7.6";
-
-  src = fetchFromGitHub {
-    owner = "radareorg";
-    repo = pname;
-    rev = version;
-    sha256 = "sha256-PnIOoWPYLK30lMmLVctihCs7GBo0rTN8yetWAr21h9w=";
-  };
+  version = "5.7.8";
 
-  nativeBuildInputs = [ meson ninja pkg-config python3 qttools wrapQtAppsHook ];
-
-  buildInputs = [ radare2 qtbase ];
+  srcs = [
+    (fetchFromGitHub rec {
+      owner = "radareorg";
+      repo = "iaito";
+      rev = version;
+      hash = "sha256-c36WLpVUnffeY6cXSEHvguo8BHyxaLAluN9hBKsQc0s=";
+      name = repo;
+    })
+    (fetchFromGitHub rec {
+      owner = "radareorg";
+      repo = "iaito-translations";
+      rev = "ab923335409fa298c39f0014588d78d926c6f3a2";
+      hash = "sha256-qkIC67a6YRwOa2Sr16Vg6If1TmAiSKUV7hw13Wxwl/w=";
+      name = repo;
+    })
+  ];
+  sourceRoot = "iaito/src";
 
   postUnpack = ''
-    sourceRoot=$sourceRoot/src
+    chmod -R u+w iaito-translations
   '';
 
-  # TODO Fix version checking and version information for r2.
-  # Version checking always fails due to values being empty strings for some
-  # reason. Meanwhile, we can safely assume that radare2's runtime and
-  # compile-time implementations are the same and remove this check.
-  patches = [ ./remove-broken-version-check.patch ];
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    python3
+    qttools
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    qtbase
+    radare2
+  ];
+
+  postBuild = ''
+    pushd ../../../iaito-translations
+    make build PREFIX=$out
+    popd
+  '';
 
   installPhase = ''
     runHook preInstall
 
     install -m755 -Dt $out/bin iaito
-    install -m644 -Dt $out/share/metainfo $src/src/org.radare.iaito.appdata.xml
-    install -m644 -Dt $out/share/applications $src/src/org.radare.iaito.desktop
-    install -m644 -Dt $out/share/pixmaps $src/src/img/iaito-o.svg
+    install -m644 -Dt $out/share/metainfo ../org.radare.iaito.appdata.xml
+    install -m644 -Dt $out/share/applications ../org.radare.iaito.desktop
+    install -m644 -Dt $out/share/pixmaps ../img/iaito-o.svg
+
+    pushd ../../../iaito-translations
+    make install PREFIX=$out -j$NIX_BUILD_CORES
+    popd
```

Comment:
you install translations into $out/share/iaito/translations but it looks like iaito does not loof for them there: 
```
Loading translations path /home/symphorien/.local/share/iaito/translations
Loading translations path /nix/store/qrd2nnpw3rj0ll9xkr06vzs1j87xibsn-qtbase-5.15.7/translations
```
and as a matter of fact on first startup only english is available

Reply:
Should work now. I guess it loaded them from `~/.local/share/iaito/translations` which I had installed when I was testing it out.

Reply:
I still get the same: 
```
Loading translations path /home/symphorien/.local/share/iaito/translations
Loading translations path /nix/store/qrd2nnpw3rj0ll9xkr06vzs1j87xibsn-qtbase-5.15.7/translations
```
I see you replaced the path to translations in ResourcePath.cpp but it's not event mentioned in this debug output...

Reply:
That's weird. I also get this message, but translations can now be selected from the drop down menu in the settings window (I double checked that I don't have them in `~/.local/share/iaito/translations`).

![image](https://user-images.githubusercontent.com/22211000/205437820-3de829cf-cad9-47ae-a188-51ffc5163cd8.png)

Reply:
I see this drop-down, it is populated as your screenshot show, but chosing french does not seem to have any effect, even after a restart.

Reply:
I'll look into this and make another PR soon.